src/transformers/modeling_mpnet.py

```diff
@@ -0,0 +1,400 @@
+# coding=utf-8
+# Copyright 2018 The Google AI Language Team Authors, The HuggingFace Inc. team, Microsoft Corporation.
+# Copyright (c) 2018, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""PyTorch BERT model. """
+
+
+import logging
+import math
+import os
+
+import torch
+from torch import nn
+from torch.nn import CrossEntropyLoss, MSELoss
+
+from .activations import gelu, gelu_new, swish
+from .configuration_bert import BertConfig
+from .file_utils import add_start_docstrings, add_start_docstrings_to_callable
+from .modeling_utils import PreTrainedModel, prune_linear_layer
+from .modeling_bert import (
+    BertEmbeddings, 
+    BertPreTrainedModel, 
+    BertIntermediate, 
+    BertOutput, 
+    BertSelfAttention,
+    BertSelfOutput, 
+    BertAttention, 
+    BertModel
+    
+)
+from .modeling_roberta import RobertaLMHead, create_position_ids_from_input_ids
```

Comment:
Our new guidelines discourage unnecessary inheritance. Please copy the useful code into your `modeling_mpnet.py` file and please do not use inheritance.